### ISSUE TITLE
This resource requires authentication

### ISSUE THREAD

**Issue by buildemloy** [bintray/gradle-bintray-plugin#151]
Execution failed for task ':empass_sdk:bintrayUpload'.
> Could not create version '1.0': HTTP/1.1 401 Unauthorized [message:This resource requires authentication]

use https://inthecheesefactory.com/blog/how-to-upload-library-to-jcenter-maven-central-as-dependency/en#_=_

**Comment by eyalbe4** [comment]
@buildemloy, it looks like the user you're using does not have permissions to create a version.

**Comment by buildemloy** [comment]
### Do not reply below this line ###

---------------------------------------------------------------------------------
Team Empass | November 03, 2016 | 09:55 -0700
---------------------------------------------------------------------------------

Thank you for writing in to us at Empass. Your query will be responded to soon.

Best regards,
Team Empass

**Comment by sathi-007** [comment]
please create your account in bintray using in the following link to avaoid this error
https://bintray.com/signup/oss
instead of 
https://bintray.com/signup

**Comment by buildemloy** [comment]
### Do not reply below this line ###

---------------------------------------------------------------------------------
Team Empass | November 22, 2016 | 03:12 -0800
---------------------------------------------------------------------------------

Thank you for writing in to us at Empass. Your query will be responded to soon.

Best regards,
Team Empass

**Comment by renhuihhh** [comment]
@sathi-007 you are right ,  when I sign up with my google or github account and it let me input company&organization name , fall in puzzle...  the account name shouldn't equals with org name , finally cause this problem

**Comment by eyalbe4** [comment]
@renhuihhh, @sathi-007 and @buildemloy,
Is there something we can do to help others avoid this issue? Thanks

**Comment by buildemloy** [comment]
### Do not reply below this line ###

---------------------------------------------------------------------------------
Team Empass | December 07, 2016 | 18:19 -0800
---------------------------------------------------------------------------------

Thank you for writing in to us at Empass. Your query will be responded to soon.

Best regards,
Team Empass

**Comment by buildemloy** [comment]
### Do not reply below this line ###

---------------------------------------------------------------------------------
Team Empass | December 07, 2016 | 18:31 -0800
---------------------------------------------------------------------------------

Thank you for writing in to us at Empass. Your query will be responded to soon.

Best regards,
Team Empass

**Comment by hirenpatelazilen** [comment]
**sathi-007 commented on 22 Nov 2016**, solution actual worked for me. Saved my day 👍

**Comment by hirenpatelazilen** [comment]
Hello, I am little bit confused with GPG public and private keys.

I have created public and private keys using command prompt, I set up those keys on bintray -> GPS signin, but I could not find reason why we are uploading our keys over there, even I have not used those keys anywhere. 

Awaiting for response.

Regards

**Comment by longerian** [comment]
**sathi-007 commented on 22 Nov 2016**  solves my problem too

**Comment by jinai6682039** [comment]
@sathi-007 thanks for save my time，its work for me！！

**Comment by NickolasHu** [comment]
@sathi-007 @longerian   @jinai6682039 How did you guys solve this problem? I have created a new account with https://bintray.com/signup/oss. but got the same issue.  [travis-ci build log](https://travis-ci.org/alibaba/Thera/jobs/216641421#L1929-L1937)

**Comment by jinai6682039** [comment]
@NickolasHu If you signup a new account on https://bintray.com/signup/oss use same github account or other account on https://bintray.com/signup , you must delete your old account first.If you did so, make sure your apikey or user are correc.

**Comment by antashev** [comment]
I had the same problem because I have differnt account name and organization name. To solve this problem use `userOrg` in bintray.gradle
example:
`
bintray {
    user = 'user name'
    key = 'key'
    pkg {
        userOrg = 'org name'
    }
}
`

**Comment by mspmax** [comment]
@antashev thank you so much mate !
